### ISSUE TITLE
Block cursor appears if you press Delete on a blank line

### ISSUE THREAD

**Issue by blakewatson** [zeit/hyper#421]
It looks as if the BEAM is still visible, but the block appears next to it.  After that happens, the cursor will overlap the character to its right.

![hyperterm-cursor](https://cloud.githubusercontent.com/assets/801425/17148266/130acd26-532c-11e6-8a46-01223ddaad5f.gif)

**Comment by BenoitAverty** [comment]
I have the same problem on linux. It happens everytime there's an impossible action, like moving after the end of the line in vim for example. It seems that the cursor changing colors is not applied to the beam cursor, as we can see on the gif : the block appears white then back to the cursor color.

When the block cursor appears in vim, closing vim brings back the beam cursor.

**Comment by rpunkfu** [comment]
Yeah, it's an issue with hterm, when cursor is un-focused it's forced to be rendered as block... I will take a look this weekend :)

**Comment by kleinfreund** [comment]
I believe I have a similar if not the same issue:

Triggering tab completions (e.g. typing `cd ` with a trailing space and hitting <kbd>Tab</kbd>) pops up the completions underneath while also forcing the cursor to render as a block.

This happens because the inline style for `background-color` of `div.cursor-node` is reset to the cursor color.

A temporary work around is forcing a transparent background color via ones hyper configuration:

```
termCSS: '.cursor-node { background-color: transparent !important; }',
```

**Comment by matheuss** [comment]
I fixed this via #1006 😄
